notes/markov.md

```diff
@@ -98,17 +151,44 @@ and we can determine the probability vector for day <span>\\(1\\)</span> by
 The probability distribution for the weather on day <span>\\(n\\)</span> is given by
 <div>\[ {\bf x}_n = {\bf M}^{n} {\bf x}_0. \]</div>
 
-## Review Questions
+## Markov Chain Example: Page Rank
+Page Rank is a straightforward algorithm which was popularized by Google Search to rank webpages. It attempts to model user behavior by assuming a random surfer continuously clicks links at random. So, the importance of a web page is determined by the probability of a random user ending up at that page. 
+
+<div class="figure"> <img src="{{ site.baseurl }}/assets/img/figs/page_rank.png" height=300 width=200/> </div>
+
+Let the above graph represent websites as nodes and outgoing links as directed eges. First, we create an adjacency matrix. 
 
-- See this [review link](/cs357/fa2020/reviews/rev-13-markov.html)
+<div>\[ {\bf A} = \begin{bmatrix}
+0 & 0 & 0 & 1 & 0 & 1 \\
+1 & 0 & 0 & 0 & 0 & 0 \\
+0 & 1 & 0 & 0 & 0 & 0 \\
+0 & 1 & 1 & 0 & 0 & 0 \\
+0 & 0 & 1 & 0 & 0 & 0 \\
+1 & 0 & 1 & 0 & 1 & 0 \\
+\end{bmatrix} \]</div>
 
-## ChangeLog
+Next, we take the accumulated weight (influence) going into a given page, and redistribute it evenly across each outgoing link. This is a Markov matrix. As before, we can perform repeated iteration on a random state vector until steady-state to find what page the user will most likely end up at. 
+
+<div>\[ {\bf A} = \begin{bmatrix}
+0 & 0 & 0 & 1.0 & 0 & 1.0 \\
+0.5 & 0 & 0 & 0 & 0 & 0 \\
+0 & 0.5 & 0 & 0 & 0 & 0 \\
+0 & 0.5 & 0.33 & 0 & 0 & 0 \\
+0 & 0 & 0.33 & 0 & 0 & 0 \\
+0.5 & 0 & 0.33 & 0 & 1.0 & 0 \\
+\end{bmatrix} \]</div>
+
```

Comment:
for a bit more intuition: sites therefore become "important" if they're linked to by other "important" sites. a site that isn't visited much linking to you doesn't help you very much, but a site that's visited a lot linking to you will significantly boost your score